### DISCUSSION TITLE
Instant burst of sequences

### DISCUSSION THREAD

**Discussion by UnityLV** [ChebanovDD/MatchSweets#12] [Q&A] [answered]
I want to make an extension that allows you to ensure that the sequences will be solved instantly at the moment of their hanging on the board, so that the sequences that need to be solved are solved without remaining on the field

![image](https://user-images.githubusercontent.com/104660372/216804408-2644a03a-75a6-4950-820e-8bd5adf4a49d.png)

My guess is to do a sequence search for solve in IBoardFillStrategy
and returns Job in Get selections

I hope that in this case the system will be cyclically called until there are no combinations left on the map to solve

What ideas do you have to implement this?

**Comment by UnityLV** [comment, marked as answer]
See the solution in my fork

![Unity_QBbi9O409Z](https://user-images.githubusercontent.com/104660372/216916752-a1925912-4422-4908-8321-d65c136b9032.gif)

**Reply by bale007** [reply]
The feature you provided is really useful!  @UnityLV 

However i couldn't find your forked repo, may i ask how you implemented this feature so that i can add it to my game as well ?

**Reply by UnityLV** [reply]
take another look

**Reply by olivernybroe** [reply]
Kinda sad that the solution requires changes in `BaseGame` :( But thanks anyways
